src/ol/format/gmlbase.js

```diff
@@ -31,6 +31,7 @@ goog.require('ol.xml');
  * ol.format.GML3.
  *
  * @constructor
+ * @abstract
```

Comment:
Ok, I see now that this is about generating warnings if this is instantiated directly.